gui/src/context/taipyReducers.ts

```diff
@@ -477,17 +477,15 @@ export const createRequestChartUpdateAction = (
     id: string | undefined,
     columns: string[],
     pageKey: string,
-    width?: number,
-    decimator?: string
+    decimatorPayload?: unknown | undefined,
```

Comment:
I thought that `decimatorPayload?: unknown` was equal to `decimator: unknown | undefined` ?

Reply:
You are right!